@kiva/kv-components/vue/KvLightbox.vue

```diff
@@ -206,6 +206,7 @@ export default {
 	},
 	emits: [
 		'lightbox-closed',
+		'lightbox-closed-clicking-outside',
```

Comment:
Should this match `lightbox-closed-click-outside` below?

Reply:
Ops, you're right!